src/cloud-sql-instance.ts

```diff
@@ -152,6 +171,14 @@ export class CloudSQLInstance {
       this.next = undefined;
       return Promise.reject('closed');
     }
+    if (this?.instanceInfo?.domainName && !this.checkDomainID) {
+      this.checkDomainID = setInterval(
+        () => {
+          this.checkDomainChanged();
+        },
+        this.checkDomainInterval || 30 * 1000
```

Comment:
nit: should we make `30 * 1000` into a global?

Reply:
I think it's ok to use 30*1000. I removed it from this line because it gets set in the constructor.